blocked-edges/4.15.7-AzureRegistryImageMigrationUserProvisioned.yaml

```diff
@@ -0,0 +1,20 @@
+to: 4.15.7
```

Comment:
I think this causes the e2e failure, 4.15.7 was tombstoned and we never released its signatures

/lgtm cancel